galaxy_ng/app/api/v3/viewsets/collection.py

```diff
@@ -240,7 +250,10 @@ def get(self, request, *args, **kwargs):
             filename=self.kwargs['filename'],
         )
 
-        response = requests.get(url, stream=True, allow_redirects=False)
+        if settings.PULP_CONTENT_BIND:
```

Comment:
Isn't it a good idea to check also if this `PULP_CONTENT_BIND` starts with `"unix:"` ?

According to gunicorn docs https://docs.gunicorn.org/en/stable/settings.html#bind 

```
A string of the form: HOST, HOST:PORT, unix:PATH, fd://FD. An IP is a valid HOST.

Changed in version 20.0: Support for fd://FD got added.
```

and pulp_installer maps it directly to that gunicorn argument https://github.com/pulp/pulp_installer/tree/master/roles/pulp_content#role-variables

```suggestion
        if settings.PULP_CONTENT_BIND and settings.PULP_CONTENT_BIND.startswith("unix:"):
```

Reply:
sounds like it must be a unix socket path and not the other forms, so yes we can check for this